manifests/server.pp

```diff
@@ -497,6 +511,9 @@
   $rcvbuf                    = undef,
   $shared_ca                 = undef,
   $crl_verify                = true,
+  $crl_auto_renew            = false,
```

Comment:
Can you add datatypes for the new params?

Reply:
Shure, if you want me to do that - i did not, since none of the params has datatypes yet...

Reply:
yeah, we want to have datatypes for all new params